### PR TITLE
chore(docs): Improve docs build time

### PR DESCRIPTION
**Pull request by talend-glorieux**
**What is the problem this PR is trying to solve?**
Docs build time takes almost 6 minutes on Travis.

**What is the chosen solution to this problem?**
Removing the uglify step from the docs building process cuts build time
by 4x. Plus it adds better debugging capabilities.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/.storybook/webpack.config.js [hunk 1]
```diff
@@ -5,38 +5,40 @@
...
-					limit: 50000,
-					mimetype: 'application/font-woff',
-					name: './fonts/[name].[ext]',
-				},
-			},
-			{
-				test: /\.scss$/,
-				loaders: ['style', 'css?sourceMap&modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]', 'postcss', 'sass'],
+module.exports = (storybookBaseConfig) => {
+	storybookBaseConfig.plugins.pop();
```
Comment (jsomsanith) [inline packages/components/.storybook/webpack.config.js]: I suppose this is to remove Uglify, but this is not clear nor futureproof.
Can you replace it with a filter + affectation ?

File: packages/containers/.storybook/webpack.config.js [hunk 2]
```diff
@@ -6,35 +6,34 @@
...
-		],
-	},
-	postcss: [
-		require('autoprefixer')({
-			browsers: ['last 2 versions']
-		})
-	],
-	sassLoader: {
+module.exports = (storybookBaseConfig) => {
+	storybookBaseConfig.plugins.pop();
```
Comment (jsomsanith) [inline packages/containers/.storybook/webpack.config.js]: Same here

File: packages/forms/.storybook/webpack.config.js [hunk 3]
```diff
@@ -8,34 +8,37 @@
...
-		],
-	},
-	postcss: [
-		require('autoprefixer')({
-			browsers: ['last 2 versions']
-		})
-	],
-	sassLoader: {
+module.exports = (storybookBaseConfig) => {
+	storybookBaseConfig.plugins.pop();
```
Comment (jsomsanith) [inline packages/forms/.storybook/webpack.config.js]: Same here

File: packages/components/.storybook/webpack.config.js [hunk 4]
```diff
@@ -5,38 +5,43 @@
 // IMPORTANT
 // When you add this file, we won't add the default configurations which is similar
 // to "React Create App". This only has babel loader to load JavaScript.
-const SASS_DATA = '@import "~bootstrap-talend-theme/src/theme/guidelines";';
+const SASS_DATA = "@import '~bootstrap-talend-theme/src/theme/guidelines';";
```
Comment (sdwvit) [inline packages/components/.storybook/webpack.config.js]: seems like unnecessary change

File: packages/forms/.storybook/webpack.config.js [hunk 5]
```diff
@@ -8,34 +8,40 @@
...
+			browsers: ['last 2 versions'],
+		}),
+	];
+
+	storybookBaseConfig.sassLoader = {
 		data: SASS_DATA,
-	},
+	};
+
+	return storybookBaseConfig;
```
Comment (sdwvit) [inline packages/forms/.storybook/webpack.config.js]: ok I see that this file is a copypaste between packages of ui/
do you consider creating commons?

File: packages/components/.storybook/webpack.config.js (hunk 4 above)
Comment (jsomsanith) [inline packages/components/.storybook/webpack.config.js]: True but I think it's ok, whatever.

File: packages/forms/.storybook/webpack.config.js (hunk 5 above)
Comment (jsomsanith) [inline packages/forms/.storybook/webpack.config.js]: :+1: Let's merge this one and make another PR for that
